display_list/effects/color_sources/dl_conical_gradient_color_source.h

```diff
@@ -35,15 +35,23 @@ class DlConicalGradientColorSource final : public DlGradientColorSourceBase {
   bool equals_(DlColorSource const& other) const override;
 
  private:
+  template <typename Colors>
```

Comment:
These constructors are so huge but have basically zero logic so I decided not to duplicate them and instead use a template to avoid any sort of bugs where they get out of sync.

Reply:
SGTM